src/components/AVPlayer/AVPlayer.js

```diff
@@ -363,14 +369,15 @@ class AVPlayer extends Component {
   }
 
   onPlay = () => {
-    const { onPlay, item } = this.props;
+    const { onPlay, item, playerPlay } = this.props;
```

Comment:
Not the best name in player. Maybe rename it to chroniclesPlay or something else?

Reply:
The event defined inside general redux/module/player.
It is used for recommendation actually not for chronicles.
I would keep the name general, maybe playerPlayAction?

What do you think?